app/models/download.rb

```diff
@@ -6,7 +6,7 @@ def initialize(work, options = {})
     @file_type = set_file_type(options.slice(:mime_type, :format))
     # TODO: Our current version of the mime-types gem doesn't include azw3, but
     # the gem cannot be updated without updating rest-client
-    @mime_type = @file_type == "azw3" ? "application/x-mobi8-ebook" : MIME::Types.type_for(@file_type).first
+    @mime_type = @file_type == "azw3" ? "application/x-mobi8-ebook" : Marcel::MimeType.for(@file_type).to_s
```

Comment:
Following from above, this can be simplified to
```suggestion
    @mime_type = Marcel::MimeType.for(extension: @file_type).to_s
```